client/src/app/(onboarding)/signup/page.tsx

```diff
@@ -118,7 +118,6 @@ export default function SignupPage() {
           </Typography>
           <TextField
             fullWidth
-            label="Email"
```

Comment:
I just reviewed how the label is being rendered. Have the label render automatically instead of manually rendering it. Manually adding it is redundant. 

<img width="488" alt="Screenshot 2024-10-07 at 8 12 17 PM" src="https://github.com/user-attachments/assets/1468bb62-ba73-4990-9dda-4c3df1fdaf4b">

Reply:
would you prefer a custom UI component then?

Reply:
It is gonna look like this as I removed the Typography
![Screenshot 2024-10-07 at 10 11 48 PM](https://github.com/user-attachments/assets/39b08bf6-f3e8-4bc3-bab7-26aa3513fff0)